packages/clone-testing/Cargo.toml

```diff
@@ -11,10 +11,15 @@ description = "Adapter for interacting with cw-multi-test via the cw-orchestrato
 
 [dependencies]
 cosmwasm-std = { workspace = true }
-cw-orch-daemon = { workspace = true }
-cw-orch-core = { workspace = true }
-cw-orch-mock = { workspace = true }
-clone-cw-multi-test = { version = "0.5", features = ["cosmwasm_1_2"] }
+
+# This package depends on published cw-orch version due to the clone-testing-cw-multi-test dependency
+cw-orch-core = { version = "2.0.0" }
```

Comment:
Can we keep these imports as workspace/path imports?

Reply:
No we can't unfortunately, because of the comment

Reply:
The exact code where this fails is here : https://github.com/AbstractSDK/cw-orchestrator/blob/221391fb8b8460cdae6aa8333f99dc2f18c95ad8/packages/clone-testing/src/core.rs#L227
The `RemoteChannel` object takes in a ChainData type

Reply:
The comment says why the dependency is there, not why it's a version vs a path dependency.

Reply:
Having both the path **and** the version allows us to: 

- Publish (because then the version is used)
- Keep it up-to-date as dev-builds use the local version

So my question is why not use the path here, in addition to the version?

Reply:
And remove the patch in the process.

Reply:
I re-iterate my point above. It's not a question of publishing, it's a question of dependencies. Without even tryong to publish, cargo check doesn't pass if the cw-orch-core version used here doesn't correspond to the version used in `clone-cw-multi-test`